Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java

```diff
@@ -116,6 +120,7 @@
                 }
             }
         });
+        //JIRA-TODO 
```

Comment:
Was there something you wanted to do here?

Reply:
Yes there was I went looking for the JIRA and got distracted, will fix that.